src/main/java/frc/team2412/robot/util/auto/AutonomousChooser.java

```diff
@@ -14,9 +14,9 @@ public interface CommandSupplier {
 
 	public AutonomousChooser() {
 		autonomousModeChooser.setDefaultOption(
-				"TopLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("TopLeaveCom"));
+				"LeftLeaveCom", () -> AutonomousTrajectories.getAutoPathByName("TopLeaveCom"));
```

Comment:
Should we also change the "TopScoreLeaveCom" option? Also (probably after GPK) it might be good to rename the files themselves, but that's low priority.